chcemvediet/templates/main/homepage/blogs/base.html

```diff
@@ -9,7 +9,7 @@
 
 
 {% block blog %}
-  <a class="chv-blog pln-not-implemented" href="#">
```

Comment:
Pls, linku otvorit v novom tabe, kedze to ide na externy web. Vratane `rel="noreferrer noopener"`.

Reply:
:ok: